pkgs/development/lisp-modules-new-obsolete/ql.nix

```diff
@@ -224,14 +220,14 @@ let
     lib.optionalAttrs (builtins.pathExists ./imported.nix)
       (import ./imported.nix { inherit (pkgs) runCommand fetchzip; pkgs = builtQlpkgs; });
 
-  builtQlpkgs = mapAttrs (n: v: build v) qlpkgs;
+  builtQlpkgs = lib.mapAttrs (n: v: build v) qlpkgs;
 
   build = pkg:
     let
       builtPkg = build-asdf-system pkg;
       withExtras = pkg //
-                   (optionalAttrs
-                     (hasAttr pkg.pname extras)
+                   (lib.optionalAttrs
+                     (lib.hasAttr pkg.pname extras)
```

Comment:
Here's the added `lib.` to see how it feels (vs. the tool-assisted extraction present in other PRs.)